src/views/Header/Header.js

```diff
@@ -179,11 +179,13 @@ class Header extends React.PureComponent<Props, State> {
   };
 
   _renderLeft(props: SceneProps): ?React.Node {
+    const interpolator =
+      this.props.leftInterpolator || HeaderStyleInterpolator.forLeft;
```

Comment:
The PR seems fairly straightforward to me, but I'd like to ask you for a change - please add `leftInterpolator` (and others) to the `static defaultProps` of the `Header` component, along with the respective default values, instead of using the `||` way. Thank you!

Reply:
Good point, thanks. Moved the defaults to `defaultProps` in a new commit.